scripts/pre_commit_linter.py

```diff
@@ -463,8 +470,238 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
+def _get_all_files():
+    """This function is used to parse the arguments entered by the user, and
+    returns the list of all files for linting and pattern checks.
+
+    Returns:
+        tuple(list(str), bool). The tuple consisting list of files for linting
+        and pattern checks and bool which checks if whether verbose
+        mode is enabled or not.
```

Comment:
This is weird. What does _get_all_files() have to do with verbose mode in the first place (looking at the name)? I recommend that you parse the args and handle all the "verbose" stuff in the main function instead.

Reply:
Done